custom/icds/tests/test_custom_expressions.py

```diff
@@ -0,0 +1,152 @@
+import uuid
+import datetime
+
+from django.test import TestCase
+from casexml.apps.case.mock import CaseStructure, CaseIndex, CaseFactory
+from casexml.apps.case.tests.util import delete_all_cases
+from corehq.apps.userreports.models import DataSourceConfiguration
+
+
+class ChildHealthExpressionTest(TestCase):
+    """
+    Integration tests for data-source configs that use the custom-expression
+    'custom.icds.ucr.expressions.child_health_indicator'
+    """
+    def setUp(self):
+        self.config = DataSourceConfiguration(
+            domain='test',
+            display_name='foo',
+            referenced_doc_type='CommCareCase',
+            table_id="ding",
+            base_item_expression={
+                'type': 'iterator',
+                'expressions': [-3, -2, -1, 0, 1, 2, 3]
+            },
+            configured_filter={
+                u'operator': u'eq',
+                u'type': u'boolean_expression',
+                u'expression': {
+                    u'type': u'property_name',
+                    u'property_name': u'type'
+                },
+                u'property_value': 'child'
+            },
+            configured_indicators=[
+                {
+                    "type": "expression",
+                    "column_id": "age_in_months",
+                    "datatype": "integer",
+                    "expression": {
+                        'type': 'child_health_indicator',
```

Comment:
As @sheelio pointed out, we can have one custom expression for all case types, instead of one for each case-type. In fact, I prefer to have one custom expression (since that will be more DRY in `ChildHealthIndicatorExpressionSpec` as well).